docs/openfaas-pro/iam/overview.md

```diff
@@ -65,36 +65,43 @@ There must be at least one registered OIDC provider for human users to authentic
 
 ## Permissions
 
-Functions:
-
-* `Function:List` - List functions
-* `Function:Get` - Get the status of functions
-* `Function:Create` - Create functions
-* `Function:Update` - Update functions
-* `Function:Delete` - Delete functions
-* `Function:Logs` - Get logs for functions
-* `Function:Scale` - Set the replica count for functions
-
-Secrets:
-
-* `Secret:List` - List secrets
-* `Secret:Create` - Create secrets
-* `Secret:Update` - Update secrets
-* `Secret:Delete` - Delete secrets
-
-> Note: (it's not possible to unable to fetch the contents of a secret via API)
-
-Namespaces:
-
-* `Namespace:List` - List namespaces
-* `Namespace:Get` - Get details of namespaces
-* `Namespace:Create` - Create namespaces
-* `Namespace:Update` - Update namespaces
-* `Namespace:Delete` - Delete namespaces
-
-System:
-
-* `System:Info` - Get system provider information
+**Functions**
+
+| Action            | Description                         | Resource scope                     | Access level |
+| ----------------- | ----------------------------------- | ---------------------------------- | ------------ |
+| `Function:Get`    | Get the status of functions         | `cluster`, `namespace`             | Read         |
+| `Function:Create` | Create functions                    | `cluster`, `namespace`             | Write        |
+| `Function:Update` | Update functions                    | `cluster`, `namespace`             | Write        |
+| `Function:Delete` | Delete functions                    | `cluster`, `namespace`             | Write        |
+| `Function:List`   | List functions                      | `cluster`, `namespace`             | List         |
+| `Function:Logs`   | Get logs for functions              | `cluster`, `namespace`             | Read         |
+| `Function:Scale`  | Set the replica count for functions | `cluster`, `namespace`             | Write        |
+| `Function:Invoke` | Invoke a function                   | `cluster`, `namespace`, `function` | Write        |
+
```

Comment:
A note on `Function:Invoke`: all functions can be invoked by default, however if they have Function Authentication enabled (link to page), then this permission is required to invoke a function.

Reply:
A note was added to explain this.